Kernel/init.cpp

```diff
@@ -246,7 +246,8 @@ void init_stage2(void*)
         }
     }
 
-    UHCIController::detect();
+    USB::UHCIController::detect();
+    USB::UHCIController::the().spawn_port_proc();
```

Comment:
Careful here, if no UHCIController is found, this causes a null deref :)

Reply:
Yeah this is going to get a _major_ cleanup soon. I've moved into the constructor for now for debug purposes.